expyfun/stimuli/tests/test_tracker.py

```diff
@@ -0,0 +1,184 @@
+import numpy as np
+from expyfun.stimuli import TrackerUD, TrackerBinom, TrackerDealer
+from expyfun import ExperimentController
+from nose.tools import assert_raises
+import matplotlib.pyplot as plt
```

Comment:
(note that in `test_viz.py` we import `pyplot` only within the test(s) that need it, not globally)